x/ccv/consumer/module.go

```diff
@@ -92,12 +93,14 @@ func (AppModuleBasic) GetQueryCmd() *cobra.Command {
 type AppModule struct {
 	AppModuleBasic
 	keeper keeper.Keeper
+	sk     ccvtypes.DemocracyStakingKeeper
 }
 
 // NewAppModule creates a new consumer module
-func NewAppModule(k keeper.Keeper) AppModule {
+func NewAppModule(k keeper.Keeper, sk ccvtypes.DemocracyStakingKeeper) AppModule {
 	return AppModule{
 		keeper: k,
+		sk:     sk,
```

Comment:
Why do we need this? The DemocracyStakingKeeper can be access via `am.keeper.stakingKeeper`

Reply:
This has to do with the capabilities model, `am.keeper.stakingKeeper` is non-exported and not accessible.